src/sagemaker_xgboost_container/algorithm_mode/hyperparameter_validation.py

```diff
@@ -203,6 +203,7 @@ def eval_metric_dep_validator(value, dependencies):
                                              required=False),
         hpv.IntegerHyperparameter(name="seed", range=hpv.Interval(min_open=-2**31, max_open=2**31-1),
                                   required=False),
+        hpv.IntegerHyperparameter(name="num_parallel_tree", range=hpv.Interval(min_closed=0), required=False)
```

Comment:
`min_closed` is inclusive of the specified value, right? Testing locally, it looks like `num_parallel_tree` must be greater than or equal to 1:
`xgboost.core.XGBoostError: value -1 for Parameter num_parallel_tree should be greater equal to 1`

Let's set `min_closed=1` instead.